services/QuillLMS/spec/controllers/admins_controller_spec.rb

```diff
@@ -284,6 +284,22 @@
         expect { subject }.to not_change(teacher, :teacher_verification_status)
       end
     end
+
+    context 'when token param is passed' do
+      let(:new_admin) { create(:admin) }
+
+      before do
+        new_admin.refresh_token!
+        # Spy on the sign_in method
+        allow(controller).to receive(:sign_in).and_call_original
```

Comment:
Given that you have an explicit `excpect` call below, I don't think you need this here.  You would need to move the `.and_call_original` from here to the end of line 298, though.  I could be wrong, in which case, just leave this as-is.